examples/golang/verify.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+export NAME=golang
+export UID
+export MANUAL=true
+
```

Comment:
```suggestion
export PORT_PROXY="${GOLANG_PORT_PROXY:-10710}"

```

the reason for this port number is a bit contrived/non-ideal - but please use that